dbt_common/exceptions/connection.py

```diff
@@ -0,0 +1,7 @@
+class ConnectionError(Exception):
```

Comment:
Is this specific to a connection to a data platform? Then it should probably live in `dbt-adapter` since that's where `connection` lives.

Reply:
This is more generic and used in `connection_exception_retry` in dbt_common.utils.connection which handles `requests.exceptions.RequestException`